src/hooks/useDelegatesInfo.ts

```diff
@@ -14,9 +15,15 @@ function useDelegatesInfo(addresses: string[]): Delegate[] {
 
   const { votingPower } = useVotingPowerBalanceList(addresses)
   const { pickedByResults } = usePickedBy(addresses)
+  const { addressesVotesTotals } = useAddressesVotesTotals(addresses)
```

Comment:
This hooks could be called at the same time if could do something similar to 
`await Promise.all( ...)`

Reply:
I'm 90% sure that calling these hooks are non-blocking since we are using `useAsyncMemo` inside them, so it shouldn't be a problem. WDYT @andyesp ?

Reply:
Yes! I think @ncomerci is right!